.ruff.toml

```diff
@@ -146,6 +146,7 @@ ignore = [
     "PTH111",  # os-path-expanduser
     "PTH112",  # os-path-isdir
     "PTH113",  # os-path-isfile
+    "PTH117",  # os-path-isabs
```

Comment:
Perhaps I'm not seeing the full context in the above conversation, but we're trying not to add ignores, instead fixing the issues and selectively applying `# noqa: ...` comments where something really needs to be special-cased. What's the situation here?

Reply:
I'm adding this ignore and does not want to be engaged in any more ruff stuff. What a follow-up PR does is not my concern.

Reply:
I'm sorry, but summary dismissal is quite rude. And unwarranted when you are editing a file. If you want to apply a specific ignore we have purposefully built a mechanism to ignore ruff rules locally. Please move the ruff ignore down to the bottom of the file in the per-subpackage section so that it effects only this subpackage and not all of Astropy.

Reply:
You don't need to engage or debate ruff, but please limit the choices in this PR to this PR and not add ignores to the global rule set.

Reply:
The rest of the PR looks quite good, though I'm not the right maintainer to review and make a final decision.

Reply:
I'm sorry, but ruff made a nightmare in this PR already, with requiring us to rebase 3 times, including a really messy conflict situation. 

And peppering the code with `#noqa` is everything but an elegant solution, especially for a rule that is phrased to be rather optional. See `Path.is_absolute() can improve readability over the os module's counterparts`

Reply:
If you have a review comment on the actual content of the PR, that is more than welcome.

Reply:
I agree that adding many noqa isn't optimal! Perhaps my suggestion of moving the ignore to the per-package section of the `.ruff.toml` is a good fix.

Reply:
> And blocking a PR on the basis of ruff, especially when it made to work, is nothing if not the rudest step. 

Requesting a change is standard practice when something requires changing. As is the case here. The ignore is fine to be moved to the per-package ignore section, but not to the global list. I'm sorry, but I don't think it's controversial to say that making something work by changing the CI absolutely merits review. Thankfully this is a very minor and easily addressable change.

Reply:
@bsipocz , I also think moving this ignore to subpackage section is a good compromise and should be low barrier for this PR?

Reply:
The serial, and apparently, never-ending low barriers on this first-contributor PR though add up and are not at all reasonable at this point. Nitpicking is not foreign to me, but this is a whole new level, all the while very little actual discussion was made about the actual content. Is that a welcoming atmosphere?

Reply:
Would you be opposed to, say, me adding a commit to address this (moving it down to subpackage level) after all the reviews are done but before merge?